src/Msg.h

```diff
@@ -64,7 +64,7 @@ struct Msg {
     }
     return Size;
   }
-  MessageMetaData const getMetaData() const { return MetaData; }
+  MessageMetaData getMetaData() const { return MetaData; }
```

Comment:
I made this function have a `const` return value due to a bug I introduced in my code and to make it impossible to re-introduce that bug. I see no reason to change that back.
```suggestion
  MessageMetaData const getMetaData() const { return MetaData; }
```

Reply:
Ok, I changed it because it didn't look useful and clang-tidy labels it as a probable mistake. Adding a comment to explain why it is there would help avoid someone making the same change in the future.

Reply:
I have added a comment to the suggestion.

Reply:
Could be const-ref?
Everywhere it is used (currently) is to access a value inside MetaData, so lifetime is not a concern.

Reply:
Sure, I do not see a problem with doing that.